aep/general/0002/aep.md

```diff
@@ -0,0 +1,64 @@
+# AEP Numbering and Organization
+
+This document describes the numbering system and general organization of AEPs.
+
+AEP numbers:
+
+- Are unique. No two AEPs share the same number.
+- Indicate the order in which they are authored.
+
+To help with organization and discovery, AEPs also have a category and a slug
+to easily discovery and refer to the AEP.
+
+## AEP Numbers
+
+### Core AEPs
+
+To uniquely identify an AEP, each one is assigned a permanent number once it
+has been merged into the aep.dev repository. The AEP number assigned **must**
+be the next available number. AEPs **must** be numbered between 1 and 9999.
+
+### Organization-specific AEPs
+
+Organizations **may** extend the AEPs with guidance that is relevant to them.
```

Comment:
I think this addresses the numbering question, but is light on details about what it means to extend the AEPs and how to do it.  Maybe this AEP doesn't need to cover those details, but it would be helpful for it to at least link to something that does.

Reply:
There's already a lot of content, so I'd rather do that incrementally if that's ok. Tracking ticket: https://github.com/aep-dev/aep.dev/issues/69.